features/admin.userstores.v1/pages/user-stores.tsx

```diff
@@ -222,6 +228,50 @@ const UserStores: FunctionComponent<UserStoresPageInterface> = (
         setFilteredUserStores(userStores);
     };
 
+    const addUserDropdownTrigger: ReactElement = (
+        <PrimaryButton
+            data-componentid={ `${ testId }-add-user-button` }
+            data-testid={ `${ testId }-add-user-button` }
+        >
+            <Icon name="add"/>
+            { t("userstores:pageLayout.list.primaryAction") }
+            <Icon name="dropdown" className="ml-3 mr-0"/>
```

Comment:
It is not recommended to use utility classes (`ml-3`, etc)

Reply:
Will be addressed in a follow up PR